language/stdlib/modules/LibraAccount.move

```diff
@@ -1102,6 +1139,29 @@ module LibraAccount {
         )
     }
 
+    /// The prologue for WriteSet transaction
+    fun writeset_prologue(
+        sender: &signer,
+        writeset_sequence_number: u64,
```

Comment:
Can we name these "txn_sequence_number" and "txn_public_key" for consistency with the other prologue functions? The account has a single sequence number shared for all kinds of transactions, right?